src/common/marker/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Created by cfaes on 24.08.2015.

```

Comment:
Is your id cfaes ? We don't give credit in our code base. We do it in changelog or other place. Is it ok for me to remove this?